kr/SUMMARY.md

```diff
@@ -0,0 +1,151 @@
+  * [Planes](airframes_plane/README.md)
+    * [Wing Wing Z-84](airframes_plane/wing_z_84.md)
+  * [VTOL](airframes_vtol/README.md)
+    * [VTOL 테스팅](airframes_vtol/testing.md)
+    * [TBS Caipiroshka](airframes_vtol/caipiroshka.md)
+  * [보트, 잠수함, 소형비행기, 로버](airframes_experimental/README.md)
+    * [Traxxas Sampede](airframes_experimental/traxxas_stampede.md)
+* [컴패니온 컴퓨터](companion_computer/README.md)
+  * [Pixhawk 계열](companion_computer/pixhawk_companion.md)
+* [DroneKit사용 로보틱스](dronekit/README.md)
+  * [DroneKit 사용법](dronekit/example.md)
+* [ROS사용 로보틱스](ros/README.md)
+  * [Linux에서 오프보드 제어](ros/offboard_control.md)
+  * [RPi에 ROS 설치](ros/raspberrypi_installation.md)
+  * [MAVROS \(ROS에서 MAVLink\)](ros/mavros_installation.md)
+  * [MAVROS 오프보드 예제](ros/mavros_offboard.md)
+  * [외부 Position Estimation](ros/external_position_estimation.md)
+  * [Gazebo Octomap](simulation/gazebo_octomap.md)
+* [센서와 액츄레이터 버스](sensor_bus/README.md)
+  * [I2C Bus](sensor_bus/i2c.md)
+    * [SF1XX lidar](sensor/sf1xx_lidar_setup.md)
+  * [UAVCAN Bus](uavcan/README.md)
+    * [UAVCAN Bootloader](uavcan/bootloader_installation.md)
+    * [UAVCAN 펌웨어 업그레이드](uavcan/node_firmware.md)
+    * [UAVCAN 설정](uavcan/node_enumeration.md)
+    * [UAVCAN 관련 노트](uavcan/notes.md)
+  * [PWM / GPIO](sensor_bus/pwm_gpio.md)
+  * [UART](sensor_bus/uart.md)
+    * [uLanding Radar](sensor/uart_ulanding_radar.md)
+* [디버깅과 고급 topics](debugging-and-advanced-topics.md)
+  * [FAQ](debug/faq.md)
+  * [시스템 콘솔](debug/system_console.md)
+  * [시스템 부트](advanced/system_startup.md)
+  * [파라미터 & 설정](advanced/parameters_and_configurations.md)
+  * [파라미터 레퍼런스](advanced/parameter_reference.md)
+  * [Autopilot 디버깅](debug/gdb_debugging.md)
+  * [센서/Topic 디버깅](debug/sensor_uorb_topic_debugging.md)
+  * [시뮬레이션 디버깅](debug/simulation_debugging.md)
+  * [디버그 값 보내기](debug/debug_values.md)
+  * [프로파일링](debug/profiling.md)
+  * [실내 / 가짜 GPS](advanced/fake_gps.md)
+  * [로깅](log/logging.md)
+  * [비행 로그 분석](log/flight_log_analysis.md)
+  * [EKF 로그 Replay](log/ekf2_log_replay.md)
+  * [시스템규모 Replay](debug/system_wide_replay.md)
+  * [Intel RealSense R200 드라이버 설치](advanced/realsense_intel_driver.md)
+  * [패롯 비밥](advanced/parrot_bebop.md)
+  * [짐벌 \(마운트\) 컨트롤 셋업](advanced/gimbal_control.md)
+  * [State Estimators 변환](advanced/switching_state_estimators.md)
+  * [외부 모듈](advanced/out_of_tree_modules.md)
+  * [ULog 파일 포맷](log/ulog_file_format.md)
+  * [라이센스](advanced/licenses.md)
+  * [텔레메트리 라디오](advanced/telemetry_radio_modem.md)
+  * [텔레메트리 Wifi](advanced/telemetry_wifi.md)
+* [소프트웨어 업데이트](software_update/README.md)
+  * [STM32 Bootloader](software_update/stm32_bootloader.md)
+* [테스팅 및 CI](test_and_ci/README.md)
+  * [Docker 컨테이너](test_and_ci/docker.md)
+  * [지속통합](test_and_ci/continous_integration.md)
+    * [Jenkins 지속통합](test_and_ci/jenkins_ci.md)
+  * [유지보수](test_and_ci/maintenance.md)
+
+
+  
+## Dronecode 바로가기
+
+* [PX4 사용자 가이드](https://docs.px4.io/en/)
+* [QGroundControl 사용자 가이드](https://docs.qgroundcontrol.com/en/)
```

Comment:
This should go to your translation (as it does exist): https://docs.qgroundcontrol.com/kr/